lib/msf/core/exploit/remote/http/wordpress/users.rb

```diff
@@ -14,7 +14,8 @@ def wordpress_user_exists?(user)
 
     return true if res and res.code == 200 and
         (res.body.to_s =~ /Incorrect password/ or
-            res.body.to_s =~ /document\.getElementById\('user_pass'\)/)
+            res.body.to_s =~ /document\.getElementById\('user_pass'\)/ or
+            res.body.to_s =~/<strong>^#{user}<\/strong> is incorrect/)
```

Comment:
Wait I think there might be a typo here. Looks like an extra ^ slipped in. Let me check this quickly.

Reply:
There we go that was the issue 👍

Reply:
Applied the fix in https://github.com/rapid7/metasploit-framework/commit/94dbe9f39e7d927b6fcd0b942b065c1b815099db before landing. Thanks again!